clients/includes/further-reading.inc

```diff
@@ -0,0 +1,4 @@
+* Use :ref:`plugins <plugins>` to customize the way HTTP requests are sent and responses
+  processed.
```

Comment:
it should be. if a client follows redirects, we end up with
inconsistency and unexpected behaviour. we try to set the adapter
clients minimal, see https://github.com/php-http/guzzle6-adapter/pull/19
for example.

ensuring this would be part of the job of the functional test suite we
talked about recently...